packages/volto/theme/themes/pastanaga/extras/main.less

```diff
@@ -17,6 +17,9 @@
 
 body {
   display: flex;
+  // RAC Popovers have problems with the default declaration coming from Semantic UI
+  // that forces `overflow-x: hidden` on the BODY.
```

Comment:
I have no idea what RAC means, but this is my guess. Please correct me if it is wrong.

```suggestion
  // React Autocomplete Component popovers have problems with the default declaration
  // in Semantic UI that forces `overflow-x: hidden` on the body.
```

Reply:
LOLOLOL

Reply:
Fixed.